src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -335,26 +335,28 @@ const SearchControls = props => {
   // Track geocode errors
   useEffect(
     () => {
-      switch (currentQuery.geocodeError) {
-        case 0:
-          break;
-        case 1:
-          recordEvent({
-            event: 'fl-get-geolocation-permission-error',
-            'error-key': '1_PERMISSION_DENIED',
-          });
-          break;
-        case 2:
-          recordEvent({
-            event: 'fl-get-geolocation-other-error',
-            'error-key': '2_POSITION_UNAVAILABLE',
-          });
-          break;
-        default:
-          recordEvent({
-            event: 'fl-get-geolocation-other-error',
-            'error-key': '3_TIMEOUT',
-          });
+      if (currentQuery?.geocodeError) {
```

Comment:
Without this check, the switch statement will hit the `default` case which logs a geolocation error. This is happening in prod when you first land on `/find-locations` before searching.

<img width="600" alt="Screenshot 2024-08-08 at 2 15 32 PM" src="https://github.com/user-attachments/assets/03488139-84f1-4e92-8d6e-2b87e58ad2d5">
<img width="500" alt="Screenshot 2024-08-08 at 1 45 42 PM" src="https://github.com/user-attachments/assets/3e2cadc6-f4c3-4a64-ab10-557f97aa6c03">